aerleon/lib/windows_advfirewall.py

```diff
@@ -75,12 +79,21 @@ def _HandleIcmpTypes(self, icmp_types, protocols):
 
         return (types, protocols)
 
-    def _HandlePorts(self, src_ports, dst_ports):
+    def _HandlePorts(
+        self, src_ports: List[Any], dst_ports: List[Union[Tuple[int, int], Any]]
```

Comment:
Anywhere Monkeytype generates `Union[..., Any]` we should try to get rid of the "Any" and figure out what the type requirement really is.